packages/app/app/components/LyricsView/index.js

```diff
@@ -18,21 +24,41 @@ export class LyricsView extends React.Component {
     lyricsStr = _.get(lyricsStr, 'type', '');
     if (lyricsStr === '') {
       lyricsStr = this.props.t('not-found');
-    }
-    return (
-      <div className={styles.lyrics_text}>
-        {lyricsStr}
-      </div>);
+	}
+	
+	//Changes by WildLeons
+	if(lyricsStr === this.props.t('not-found')) {
```

Comment:
Checking if lyrics are empty should be done in a more generic way than comparing it to the internationalised string

Reply:
Also instead of using if/else, rendering can use && or the ternary operator.
```typescript
{ condition && <Component /> }
{ condition ? <Component/> : <DifferentComponent/> }
```